### ISSUE TITLE
Redis OCF is not working for slaves - Unable to connect to MASTER: Resource temporarily unavailable

### ISSUE THREAD

**Issue by lejeczek** [ClusterLabs/pcs#288]
Hi guys.

I try to follow OCF_HEARTBEAT_REDIS man page which suggest setting Redis will be simple.
I do:
`-> $ pcs resource create redis ocf:heartbeat:redis bin=/usr/bin/redis-server client_bin=/usr/bin/redis-cli config=/etc/redis.conf rundir=/run/redis user=redis op monitor interval="1s" role="Master" timeout="5s" on-fail="restart" op monitor interval="2s" role="Slave" timeout="5s" on-fail="restart" promotable`

Only three nodes and each with "replicaof" absent in redis.conf but with "masterauth" utilized (since man pages does not mention relevant bits of redis.conf at all) 

Master starts on the node cluster chooses:
...
`1281452:M 08 May 2021 09:09:46.423 * MASTER MODE enabled (user request from 'id=15 addr=/run/redis/redis.sock:0 fd=9 name= age=0 idle=0 flags=U db=0 sub=0 psub=0 multi=-1 qbuf=34 qbuf-free=32734 argv-mem=12 obl=0 oll=0 omem=0 tot-mem=61476 events=r cmd=slaveof user=default'`

but slaves, though cluster reports:
...
`* Clone Set: redis-clone [redis] (promotable):`
    `* redis	(ocf::heartbeat:redis):	 Slave 10.1.1.224`
    `* redis	(ocf::heartbeat:redis):	 Master 10.1.1.223`
    `* redis	(ocf::heartbeat:redis):	 Slave 10.1.1.225`

fail with this in the log:
...
`# Unable to connect to MASTER: Resource temporarily unavailable`
...
Redis cluster works as expected outside of pacemaker/pcs.

pacemaker-2.0.5-9.el8.x86_64
redis-6.0.9-2.module_el8.4.0+580+92c8284e.x86_64
redis-doc-6.0.9-2.module_el8.4.0+580+92c8284e.noarch
pcs-0.10.8-1.el8.x86_64

regards, L

**Comment by tomjelinek** [comment]
Hi @lejeczek,

Sorry, I don't have any experience with Redis and I think it applies to the rest of pcs team as well. Your issue doesn't seem to be pcs related, based on information you provided and the fact that pacemaker was able to start the resource. Try asking your question at users@clusterlabs.org mailing list to reach broader audience.

**Comment by lejeczek** [comment]
Neither am I but I think RA can take some blame here. The issue certainly be easy to reproduce.
Like I said - same Redis cluster works outside of PCS, stop such cluster, create redis resource and seconds later Redis slaves fail with "Unable to connect to MASTER"
I wasn't sure where to report on specific RA.
What is more interesting and puzzling - on such PCS/RA managed cluster for redis slave do:

`-> $ pcs resource ban redis-clone 10.1.1.224; pcs resource clear redis-clone 10.1.1.224`

and that node connects to the master in that instant.
many thanks, L.
ps. prior I came here I did emailed the list but got no feedback.
